server/app/views/applicant/NavigationFragment.html

```diff
@@ -233,7 +237,7 @@
   <div class="usa-footer__primary-section">
     <div class="usa-footer__primary-container grid-row">
       <div class="mobile-lg:grid-col-8">
-        <nav class="usa-footer__nav" aria-label="Footer navigation,">
+        <nav class="usa-footer__nav" th:aria-label="#{label.footerNav}">
```

Comment:
Blocking: We can remove this aria-label too. I just tested it and VoiceOver already announces "navigation" for a <nav> element such as this one

Reply:
Just to confirm I'm understanding you correctly, Do we want to remove `th:aria-label="#{label.footerNav}">` and `th:aria-label="#{label.officialWebsiteOfUsa}"` as well as the `aria-hidden="true"` on line 67? or just the aria-hidden?